src/vm/runners/builtin_runner.rs

```diff
@@ -547,27 +546,9 @@ mod tests {
 
     #[test]
     fn deduce_memory_cell_pedersen_for_preset_memory_valid() {
-        let mut memory = Memory::new();
+        let memory = memory![((0, 3), 32), ((0, 4), 72), ((0, 5), 0)]; //Memory::new();
```

Comment:
remove the commented code?

Reply:
siryessir https://github.com/lambdaclass/cairo-rs/pull/407/commits/17370f51df84fe560b323bc6152262fda03b5139